imageSim/SBmodels.py

```diff
@@ -54,8 +55,8 @@ def setPars(self):
             self.__setattr__(key,self.vmap[key].value)
 
 
-class Sersic(SBModel,SBProfiles.Sersic):
-    _baseProfile = SBProfiles.Sersic
+class Sersic(SBModel, _Sersic):
```

Comment:
This might be a bug: I don't know why Tom uses `SBProfiles.Sersic` instead of `profiles._Sersic` here. I ended up avoiding `SBProfiles` entirely, because of the name conflicts: `Sersic` is defined in both `SBProfiles.py` and in `SBModels.py` :-/